PoseEstimation-TFLiteSwift/TFLiteImageInterpreter.swift

```diff
@@ -163,7 +154,11 @@ extension TFLiteImageInterpreter {
         init(modelName: String, threadCount: Int = 1, accelerator: Accelerator = .metal, isQuantized: Bool = false, inputWidth: Int, inputHeight: Int, isGrayScale: Bool = false, isNormalized: Bool = false) {
             self.modelName = modelName
             self.threadCount = threadCount
+            #if targetEnvironment(simulator)
+            self.accelerator = .cpu
+            #else
             self.accelerator = accelerator
+            #endif
```

Comment:
In this line, https://github.com/tucan9389/PoseEstimation-TFLiteSwift/issues/49 will be solved.